src/OpenTelemetry.AutoInstrumentation/Instrumentations/StackExchangeRedis/StackExchangeRedisConstants.cs

```diff
@@ -0,0 +1,37 @@
+// <copyright file="StackExchangeRedisConstants.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+#if NETCOREAPP3_1_OR_GREATER
+
+namespace OpenTelemetry.AutoInstrumentation.Instrumentations.StackExchangeRedis;
+
+internal static class StackExchangeRedisConstants
+{
+    public const string AssemblyName = "StackExchange.Redis";
+
+    public const string MinimumVersion = "2.0.0"; // 2.1.58, but dll uses 2.0.0
```

Comment:
Have you tested the behavior when `StackExchange.Redis` version is `< 2.1.58`?

Reply:
I think you can also consider creating an issue in StackExchange.Redis`https://github.com/StackExchange/StackExchange.Redis
 See: https://github.com/StackExchange/StackExchange.Redis/blob/74f4c9a0d39b822f67e50e851d764f68d540f304/tests/ConsoleTest/Program.cs#L47

Reply:
https://github.com/StackExchange/StackExchange.Redis/pull/2197

Reply:
To echo my comment on the issue:

>This is not accidental, and is a pattern pretty common among a wide range of libraries. The reality is: having a granular assembly-version places a very large burden on consumers, in particular (but not quite exclusively) on .NET Framework - mostly due to assembly binding redirect problems. Having this break constantly creates unnecessary friction, which is often avoided by pinning the assembly version to majors. Or at a push: major+minor.
> 
> My suggestion here would be for OpenTelemetry AutoIntrumention should be tweaked to be a little sympathetic to this intent, perhaps considering other attributes.

Reply:
That assembly version pattern is common, and is actually listed in some [.net versioning guidance documentation](https://docs.microsoft.com/en-us/dotnet/standard/library-guidance/versioning).